tests/utils/test_string_utils.py

```diff
@@ -30,3 +30,9 @@ def test_is_string_type():
     assert not is_string_type({"test": "string"})
     assert not is_string_type(12)
     assert not is_string_type(12.7)
+
+
+def test_mslex_quote():
+    assert mslex_quote("abc") == "abc"
+    assert mslex_quote("a b c") == '"a b c"'
+    assert mslex_quote("C:\\file\\to\\file") == "C:\\file\\to\\file"
```

Comment:
A minimum test to ensure `C:\\file\\to\\file` is not converted to `'C:\\file\\to\\file'`.